plugins/toolbox/README.md

```diff
@@ -0,0 +1,9 @@
+# toolbox
+
+This plugins adds `toolbox_prompt_info()` function. It shows the toolbox indicator ⬢ if you are running in a container and nothing if not.
+
+It can be useful to customize themes. For example:
```

Comment:
Hmm I see you took the advice of using `kubectx` plugin as reference. That one's README should be fixed 😁 I like something more like the terraform plugin which is a little bit more verbose.